rust/src/ftp/mod.rs

```diff
@@ -55,6 +55,42 @@ named!(pub ftp_pasv_response<u16>,
         )
 );
 
+// PORT 192,168,0,13,234,10
+named!(pub ftp_active_port<u16>,
+       do_parse!(
+            tag!("PORT") >>
+            ws!(digit) >> tag!(",") >> digit >> tag!(",") >>
+            digit >> tag!(",") >> digit >> tag!(",") >>
+            part1: verify!(getu16, |v| v <= std::u8::MAX as u16) >>
+            tag!(",") >>
+            part2: verify!(getu16, |v| v <= std::u8::MAX as u16) >>
+            tag!("\0") >>
```

Comment:
Is this \0 on the wire? If not, it should not be parsed here.

Reply:
So removing the 'tag("\0")' condition and not passing +1 doesn't work?

Reply:
That's correct. The passive equivalents of these functions terminate with a `)`